src/shared/common/firebaseFunctions.js

```diff
@@ -126,6 +131,10 @@ export const mapStateToPropsForGroups =
             const groupsToPickFrom = groupsAvailable.filter(
                 g => !groupsMapped.includes(g),
             );
+            if (groupsToPickFrom.length === 1) {
```

Comment:
Is there any possibility that the length of the list drops from 2 to 0 without ever being 1? I don't think so, but just double checking :)

Reply:
good catch. I will change to <= 1